service/resource/endpoint/create.go

```diff
@@ -38,7 +38,7 @@ func (r *Resource) GetCreateState(ctx context.Context, obj, currentState, desire
 	return createState, nil
 }
 
-func (r *Resource) ProcessCreateState(ctx context.Context, obj, createState interface{}) error {
+func (r *Resource) ApplyCreateChange(ctx context.Context, obj, createState interface{}) error {
```

Comment:
Lets please sort the methods alphabetically, while public methods are at the top and private ones at the bottom.

Reply:
Fixed.